dali/image/generic_image.cc

```diff
@@ -49,18 +49,19 @@ DALIError_t GetBMPImageDims(const uint8 *bmp, int size, int *h, int *w) {
     DALIError_t ret = DALIError;
```

Comment:
BMP fix is a separate patch. Please remove it from this PR.

Reply:
It is from #156 - but this is required by this PR to work.